icecreamshop/src/photoUrls/PhotoUrls.js

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Acest fisier nu aduce nicio functionalitate codului. Il poti pune (daca vrei) in fisierul `README.md` sau poti crea o pagina noua Wiki (https://github.com/SummerSchoolAcademy/IceCream-Backend/wiki).